stock_scanner_web/controllers/main.py

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Angel Moya <http://angelmoya.es>
+# Copyright 2016 Eficent Business and IT Consulting Services, S.L.
+# <http://www.eficent.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import http
+from openerp.http import request
+from openerp import _
+
+
+def allowed_hardware(user, t_num):
+    allowed = False
+    for hardware in user.scanner_hardware_ids:
+        if t_num == hardware.code:
+            allowed = True
+    return allowed
+
+
+class ScannerWeb(http.Controller):
+    @http.route([
+        '/scanner_call',
```

Comment:
I dunno if we should add into a namespace that isn't owned by this module. It is convention to use `/module_name/*` in order to maintain namespace isolation of modules